<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lvlWeight">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Weight :</string>
    </property>
   </widget>
   <widget class="QLabel" name="lvlHeader">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>GravitYou</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnVenus">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>160</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	font: 700 9pt &quot;Segoe UI&quot;;

borde: none;
background-color: rgb(197, 193, 192);
color: rgb(102, 102, 102);
}

QPushButton:Hover{
background-color:white;
}</string>
    </property>
    <property name="text">
     <string>Venus</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEarth">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>160</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	font: 700 9pt &quot;Segoe UI&quot;;
background-color:  rgb(75, 132, 255);
color:  rgb(124, 224, 255);
}

QPushButton:Hover{
background-color:white;
}</string>
    </property>
    <property name="text">
     <string>Earth</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMercury">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>160</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	font: 700 9pt &quot;Segoe UI&quot;;
background-color:rgb(255, 180, 105);
color: rgb(255, 15, 15);
}

QPushButton:Hover{
background-color:white;
}</string>
    </property>
    <property name="text">
     <string>Mercury</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSaturn">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>220</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	font: 700 9pt &quot;Segoe UI&quot;;
background-color:rgb(255, 180, 105);
color: rgb(255, 15, 15);
}

QPushButton:Hover{
background-color:white;
}</string>
    </property>
    <property name="text">
     <string>Saturn</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMars">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>220</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	font: 700 9pt &quot;Segoe UI&quot;;
background-color:rgb(255, 180, 105);
color: rgb(255, 15, 15);
}

QPushButton:Hover{
background-color:white;
}</string>
    </property>
    <property name="text">
     <string>Mars</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnJupiter">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>220</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	font: 700 9pt &quot;Segoe UI&quot;;
background-color:rgb(255, 180, 105);
color: rgb(255, 15, 15);
}

QPushButton:Hover{
background-color:white;
}</string>
    </property>
    <property name="text">
     <string>Jupiter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnUrenus">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>280</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	font: 700 9pt &quot;Segoe UI&quot;;
background-color:rgb(255, 180, 105);
color: rgb(255, 15, 15);
}

QPushButton:Hover{
background-color:white;
}</string>
    </property>
    <property name="text">
     <string>Uranus</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnNeptune">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>280</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	font: 700 9pt &quot;Segoe UI&quot;;
background-color:rgb(255, 180, 105);
color: rgb(255, 15, 15);
}

QPushButton:Hover{
background-color:white;
}</string>
    </property>
    <property name="text">
     <string>Neptune</string>
    </property>
   </widget>
   <widget class="QLabel" name="lvlSubHead">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Gravitational Force on Different Planets (GravitYou)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBlackhole">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>280</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	font: 700 9pt &quot;Segoe UI&quot;;
background-color:rgb(66, 25, 11);
color: rgb(198, 200, 199);
}

QPushButton:Hover{
background-color:black;
}</string>
    </property>
    <property name="text">
     <string>Blackhole</string>
    </property>
   </widget>
   <widget class="QLabel" name="lETail">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>470</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>A scientific tool for Space Enthusiast</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lEGravity">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>330</y>
      <width>151</width>
      <height>24</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lEInput">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>90</y>
      <width>211</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lEInfo">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>370</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="maxLength">
     <number>350</number>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="cursorMoveStyle">
     <enum>Qt::VisualMoveStyle</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>330</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Gravitaional Force :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>90</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>kg</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lEInfo_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>410</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="maxLength">
     <number>350</number>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="cursorMoveStyle">
     <enum>Qt::VisualMoveStyle</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>130</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../Pictures/Planets icon/globeSmall.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>130</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../Pictures/Planets icon/mercurySmall.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>189</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../Pictures/Planets icon/marsSmall.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>189</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../Pictures/Planets icon/jupiterSmall.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>190</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../Pictures/Planets icon/saturnSmall.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>250</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../Pictures/Planets icon/blackholeSmall.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>250</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../Pictures/Planets icon/neptuneSmall.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>250</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../Pictures/Planets icon/uranusSmall.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>130</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../Pictures/Planets icon/venusSmall.png</pixmap>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
